tanagra-aou-utils/run_tanagra_server.sh

```diff
@@ -30,11 +30,11 @@ export TANAGRA_FEATURE_ARTIFACT_STORAGE_ENABLED=true
 export TANAGRA_AUTH_IAP_GKE_JWT=false
 
 if [[ ${disableAuthChecks} ]]; then
-  echo "Disabling auth checks."
+  echo "Disabling AuthZ"
   export TANAGRA_AUTH_DISABLE_CHECKS=true
   export TANAGRA_AUTH_BEARER_TOKEN=false
 else
-  echo "Enabling auth checks."
+  echo "Enabling AuthZ"
```

Comment:
Please change to Authentication (AuthN)
These properties are for Authentication (AuthN) bearer-token to user-lookup.
Authorization (AuthZ) is governed by [TANAGRA_ACCESS_CONTROL_MODEL](https://github.com/all-of-us/workbench/blob/adbe6e74463b4d212c96f1af3bd75e2115f748d7/tanagra-aou-utils/appengine/tanagra_env_variables.yaml#L10-L10)
`OPEN_ACCESS` means no Authorization (AuthZ)

Reply:
Done.